### DISCUSSION TITLE
Review Pull Request: Step 5

### DISCUSSION THREAD

**Discussion by tmcdermottgs** [skills/.github#272] [Q&A] [answered]
I'm stuck on step 5 of the Review Pull Request course. I added a comment in step 4. In step 5 it says to click "Commit suggestion" but I do not see that button anywhere I look on any of the pages. I only see "Resolve conversation." Am I missing something?

**Comment by sinsukehlab** [comment, marked as answer]
You haven't suggested a change in Step 4.



## Step 4: Suggest changes

_Nice work reviewing that pull request :sparkles:_

Now that you have explored the different ways you can review a pull request it is time to learn how to use _suggest changes_.

**What is _suggest changes_?**: This feature enables you to recommend a change to a pull request that the author can commit with the push of a button.

### :keyboard: Activity: Suggest changes

1. On the pull request, click **Files changed**.
1. Find the `index.html` changes.
1. Hover your cursor next to the line numbers on the left side of the page.
1. Click the blue plus icon.
1. After the comment form appears, click the **Add a suggestion** button. <br>
   ![add-a-suggestion-button](https://user-images.githubusercontent.com/97056108/184449714-61e8ee51-824a-48c1-9436-2dfd67f2c070.png)
1. Edit the suggestion.
1. Click **Add a single comment**.
1. Wait about 20 seconds then refresh this page (the one you're following instructions from). [GitHub Actions](https://docs.github.com/en/actions) will automatically update to the next step.

**Reply by tmcdermottgs** [reply]
Ah, thanks, I missed the "Add a suggestion" box and just typed a comment.

**Reply by sinsukehlab** [reply]
I'm glad that I could be of help.
